src/main/java/Calculater.java

```diff
@@ -0,0 +1,166 @@
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+public class Calculater {
+    private Deque<String> stack = new ArrayDeque<>();
+    private Deque<String> nstack = new ArrayDeque<>();
+    private int prev = 0;
+
+    public int calculate(String s) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'calculate(String)' has an NPath complexity of 366](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4535407)